_posts/2020-Q1-open-source-report.markdown

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: "2020 Q1 open-source-report"
+date: 2020-04-09 20:12:12
```

Comment:
@bronzdoc You'll need to update the date here when all comments have been addressed.